iceoryx_binding_c/include/iceoryx_binding_c/types.h

```diff
@@ -33,13 +33,13 @@
     (144 + numberOfAttachments * 168 - 8 * (((numberOfAttachments + 1) / 2) - 1))
 #elif defined(_WIN32)
 #define CALCULATE_STORAGE_SIZE_FOR_LISTENER(numberOfAttachments)                                                       \
-    (168 + numberOfAttachments * 176 - 8 * (((numberOfAttachments + 1) / 2) - 1))
+    (168 + numberOfAttachments * 192 - 8 * (((numberOfAttachments + 1) / 2) - 1))
```

Comment:
Still having issues to understand the things here. Can you describe in detail how the variables here are connected with the variables names in line 29?

Reply:
I do not know how this formula was derived but I guess is something like try it for n = 1, n = 2, n = 3 and observe the differences in `sizeof`. Extrapolate from there ..., check other values.

Maybe we should add a safety multiplicative and/or additive constant X*(1+eps) + C. Maybe this is already in the 168 and (partly 192), who knows.

I would prefer a simpler formula at least like A*numberOffAttachments + B. Why is this not possible?

Reply:
These are just magic numbers and we could also add fixed numbers instead of the formula. For the Listener it is more complicated since it contains multiple member which are dependent on the number of attachments.

There are tests that ensure that the storage will fit the objects. We could as well comment this out and add plain magic numbers like with the other storage types.

Reply:
A quick calculation shows (unless I made a mistake) that we have (with n = numberOfAttachments):

formula in code snippet <= (168 +8) + (192-4) * n = 176 + 188*n

This is only with a slight gap compared to the previous formula. Fee free to check this.

Reply:
My thought was maybe we have to change the size again quickly and then we can uncomment the cmake option and use this. But we could replace this also with hardcoded numbers.
Then we have the fun to recalculate the size of the waitset and the listener and a change in the waitset/listener capacity would force a new iceoryx release upon us  - which is fun.